### ISSUE TITLE
Invalid default version

### ISSUE THREAD

**Issue by 8tomat8** [simonmartyr/picnic-api#9]
In my tests I keep getting error
```
picnic-api: produced an error with code [BAD_REQUEST] and message [No enum constant com.picnic.meta.ApiVersion.API_17]
```

Adding `WithVersion("15")` fixes the issue. 

https://github.com/MRVDH/picnic-api also uses 15 as default version.

**Comment by simonmartyr** [comment]
Hi @8tomat8 ,

Good point, I agree. Going to switch the default to 15.

**Comment by simonmartyr** [comment]
Resolved in [V1.2.2](https://github.com/simonmartyr/picnic-api/releases/tag/v1.2.2)
